packages/cli/src/tools/releaseChecker/printNewRelease.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @flow
+ */
+import boxen from 'boxen';
+import chalk from 'chalk';
+import type {Release} from './getLatestRelease';
+
+/**
+ * Notifies the user that a newer version of React Native is available.
+ */
+export default function(latestRelease: Release, currentVersion: string) {
+  console.log(
+    boxen(
+      'A newer release of React Native is available!\n\n' +
```

Comment:
I'm not a fan of `boxen`, it reminds me bad times when Bower was prompting me all the time to use new version. I can sometimes see npm doing so, but I use it so rarely that I don't really notice it. The memories still remain :D

Maybe we can just prompt a small info about new version + link to changelog? Similar how Yarn does it

Reply:
I tend to agree, actually. I kinda feel like only outdated and old packages use boxen.